src/renderer/components/dialogs/CreateChat.tsx

```diff
@@ -334,6 +334,18 @@ export function AddMemberInnerDialog({
     .map(([contactId, _contact]) => contactId)
 
   const [contactIdsToAdd, setContactIdsToAdd] = useState<JsonContact[]>([])
+  const [{ queryStrIsValidEmail }, updateContacts] = useContactsNew(
+    C.DC_GCL_ADD_SELF,
+    ''
+  )
+  const [{}, onSearchChangeNewContact] = useContactSearch(updateContacts)
+
+  const onSearchChangeValidation = (query: ChangeEvent<HTMLInputElement>) => {
+    if (searchContacts.size === 0) {
+      onSearchChangeNewContact(query)
+    }
```

Comment:
For what is this code needed? It looks like you don't use the useContactSearch() state that you defined in line 341?

Reply:
I had to use the useContactSearch handler because it does the input validation (if it's a valid email). I figured out it was better to validate the input using existing methods than adding a new regex